src/Android/Xamarin.Android/Samples/Network Analysis/FindServiceAreasForMultipleFacilities/readme.md

```diff
@@ -0,0 +1,38 @@
+# Find service areas for multiple facilities
+
+Find the service areas of several facilities from a feature service.
+
+![](FindServiceAreasForMultipleFacilities.jpg)
+
+## Use case
+
+A service area is a region which can be accessed from a facility as limited by one or more factors, such as travel time, distance, or cost. When analyzing the service area of multiple facilities, this workflow can be used to identify gaps in service area coverage, or significant overlaps, helping to optimize the distribution of facilities. For example, a city's health service may identify areas of a city that can be effectively accessed from particular hospitals, and with this optimize distribution of staff and resources.
+
+## How to use the sample
+
+Click 'find service areas' to calculate and display the service area of each facility on the map. The polygons displayed around each facility represent the service area, with colors representing a cutoff for impedance (in this case, travel time).
```

Comment:
I think the travel time value should be explicit in the description and/or the sample UI ("Find 5 minute drive area", eg).